packages/app/src/crash-dump/actions/send-crash-dump-data.action.ts

```diff
@@ -45,8 +46,14 @@ export const sendCrashDumpData = createAsyncThunk(
       attachments,
     }
 
+    const logData: Omit<FreshdeskTicketData, "type" | "attachments"> = {
+      subject: "Error - Crash dump - logs",
+      serialNumber: state.device.data.serialNumber,
+    }
+
     try {
       await createFreshdeskTicket(data)
+      await sendTicketRequest(logData)
```

Comment:
In my opinion, a single ticket will be easier to maintenance and pair with a crash dump.

What do you think about copy/past the logic about download & zipped logs here or some helper function? To avoid too much code redundancy. Downloading logs logic (mc, pure, updater) could be moved to a helper function.